tools/tcoffee/.shed.yml

```diff
@@ -0,0 +1,12 @@
+name: tcoffee
+owner: bgruening
+description: A collection of tools for Computing, Evaluating and Manipulating Multiple Alignments of DNA, RNA, Protein Sequences and Structures
+homepage_url: http://www.tcoffee.org/Projects/tcoffee/index.html
+long_description: T-Coffee is a multiple sequence alignment package. You can use T-Coffee to align sequences or to combine the output of your favorite alignment methods (Clustal, Mafft, Probcons, Muscle...) into one unique alignment.
+remote_repository_url: https://github.com/bgruening/galaxytools/tree/master/tools/tcoffee
+type: unrestricted
+categories:
+- Sequence Alignment
+- DNA
+- RNA
```

Comment:
You can only use categories from https://toolshed.g2.bx.psu.edu
* Sequence Analysis